app/src/androidTest/java/com/github/fribourgsdp/radio/mockimplementations/MockFileSystem.kt

```diff
@@ -0,0 +1,27 @@
+package com.github.fribourgsdp.radio.mockimplementations
+
+import android.content.Context
+import com.github.fribourgsdp.radio.FileSystem
+
+var data: String = ""
```

Comment:
Why don't you put this inside the class directly?

Reply:
Since it is a "filesystem", and my implementation generates a new file system instance every time we need to read or write due to backwards compatibility with the context signature, I need it to be a global variable so that the string is the same between filesystem instances

Reply:
Okay